powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/ActivationRecordEntity.java

```diff
@@ -667,6 +669,16 @@ public List<ActivationHistoryEntity> getActivationHistory() {
         return activationHistory;
     }
 
+    @PostLoad
+    public void setDefaultProtocol() {
```

Comment:
This is a fix for null values in the DB. But the null values are updated in the liquibase script.